example_impl/example_testsuite/example_test_1.go

```diff
@@ -0,0 +1,53 @@
+package example_testsuite
```

Comment:
can we rename these to more descriptive titles of what the tests are doing rather than "1" and "2"?

Reply:
Done in the PR comments PR